pkg/schemas/generator.go

```diff
@@ -105,11 +137,125 @@ func (g Generator) Generate(ctx *genall.GenerationContext) error {
 			documents[documentName] = document
 		}
 		document.Definitions[context.definitionNameFor(documentName, typeIdent)] = typeSchema
```

Comment:
Does this mean that you will _also_ generate `external.json` if all we have is something with a `fybrik:validation:object` marker?

Reply:
All the types that are in packages without the "schema" marker will stored in `external.json`.